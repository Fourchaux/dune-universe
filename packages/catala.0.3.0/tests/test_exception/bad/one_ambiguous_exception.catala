## [Test]

```catala
new scope A:
  param x content int
  param y content int

scope A:
  def x := 0

  exception
  def x := 1

  def y := 2

  def y := 4

  exception
  def y := 3
```
