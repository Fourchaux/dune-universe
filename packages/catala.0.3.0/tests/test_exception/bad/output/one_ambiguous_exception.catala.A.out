[ERROR] This exception can refer to several definitions. Try using labels to disambiguate
[ERROR] 
[ERROR]   --> test_exception/bad/one_ambiguous_exception.catala
[ERROR]    | 
[ERROR] 16 |   def y := 4
[ERROR]    |            
[ERROR] 17 | 
[ERROR]    | 
[ERROR] 18 |   exception
[ERROR]    |   ^^^^^^^^^
[ERROR] 19 |   def y := 3
[ERROR]    |   ^^^^^^^^^^^
[ERROR]    + Test
